jwt.go

```diff
@@ -19,13 +19,193 @@ var TimeFunc = time.Now
 // Header of the token (such as `kid`) to identify which key to use.
 type Keyfunc func(*Token) (interface{}, error)
 
+// For a type to be a Claims object, it must just have a Valid method that determines
+// if the token is invalid for any supported reason
+type Claimer interface {
```

Comment:
`Claimer` should have a method `Claim`.  Why not just make this `Claims` and call the struct type `StandardClaims` or some such?